### PR TITLE
Users with tasks list

### PR DESCRIPTION
**Pull request by miroslavrs81**

### CODE REVIEW

File: src/app/standup/standup.controller.ts [hunk 1]
```diff
@@ -57,6 +58,7 @@ export class StandupController {
     userId?: number;
     isStandupInProgress: boolean;
     isLastMember: boolean;
+    usersTasks: User[] | Task[];
```
Comment (milosst94) [inline src/app/standup/standup.controller.ts]: Was this Task [ ] needed?

File: src/app/workspace/dto/create-workspace.dto.ts [hunk 2]
```diff
@@ -1,9 +1,12 @@
 import { ApiProperty } from '@nestjs/swagger';
-import { IsString } from 'class-validator';
+import { IsString, Length } from 'class-validator';
 
 export class CreateWorkspaceDto {
   @ApiProperty()
   @IsString()
+  @Length(1, undefined, {
```
Comment (milosst94) [inline src/app/workspace/dto/create-workspace.dto.ts]: This undefined means that we have no boundaries. Does limit of 255 characters have more sense in this context?

File: src/app/workspace/workspace.service.ts [hunk 3]
```diff
@@ -243,6 +244,9 @@ export class WorkspaceService {
       .execute();
     const workspace = await this.workspaceRepository.findOneBy({ id });
 
+    if (workspace.isActive === true) {
+      throw new BadRequestException(returnMessages.WorkspaceIsActive);
```
Comment (milosst94) [inline src/app/workspace/workspace.service.ts]: if isActive is set on false than restore doesn't make sense, the order of validations should be if workspace exists then if it isActive (restore has sense on something that is only soft deleted).

File: src/app/workspace/dto/create-workspace.dto.ts [hunk 4]
```diff
@@ -1,9 +1,12 @@
...
+    message: 'projectName must have at least 1 character',
```
Comment (milosst94) [inline src/app/workspace/dto/create-workspace.dto.ts]: This message should be  deleted or placed in error message mapper

**Review by Branimir — APPROVED** [review]

**Review by mickotomic — APPROVED** [review]

**Pull request merged**